esp-hal/src/spi/master.rs

```diff
@@ -2483,34 +2413,22 @@ pub trait Instance: private::Sealed {
     fn init(&mut self) {
         let reg_block = self.register_block();
         reg_block.user().modify(|_, w| {
-            w.usr_miso_highpart()
-                .clear_bit()
-                .usr_miso_highpart()
-                .clear_bit()
-                .doutdin()
-                .set_bit()
-                .usr_miso()
-                .set_bit()
-                .usr_mosi()
-                .set_bit()
-                .cs_hold()
-                .set_bit()
-                .usr_dummy_idle()
-                .set_bit()
-                .usr_addr()
-                .clear_bit()
-                .usr_command()
-                .clear_bit()
+            w.usr_miso_highpart().clear_bit();
+            w.usr_miso_highpart().clear_bit();
```

Comment:
```suggestion
            w.usr_mosi_highpart().clear_bit();
```

Reply:
Nice, this indicates an advantage of this style :D